### ISSUE TITLE
Complex Types 

### ISSUE THREAD

**Issue by HaroonSaid** [tomasfabian/Kafka.DotNet.ksqlDB#4]
Hi Tomas

Is there a way to create a table, using a model that has arrays, my Kafka topic looks like 
The approach that I took, was to create a `CREATE TYPE` and `CREATE TABLE` using the cli. 
However, it would be nice if I can do it thru the c# client

```
    public abstract class AbstractProducerClass
    {
        public string Id { get; set; }
        public string Key { get; set; }
    }
    public class EventCategory
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
    }
    // The Topic that needs the table
    public class ElasticSearchEvent : AbstractProducerClass
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public long EventId { get; set; }
        public IEnumerable<EventCategory> EventCategories { get; set; }
    }
}
```
Thanks

**Comment by HaroonSaid** [comment]
Thanks for the guidance.

**Comment by tomasfabian** [comment]
hi @HaroonSaid,
could you please try out the above requested feature? 

[CreateTypeAsync](https://github.com/tomasfabian/Kafka.DotNet.ksqlDB/blob/main/README.md#createtypeasync-v160)

```
Install-Package Kafka.DotNet.ksqlDB -Version 1.6.0-rc.1
```

```C#
var  httpResponseMessage = await restApiClient.CreateTypeAsync<EventCategory>();
```

**Comment by tomasfabian** [comment]
I released v1.6.0
